Test failure is related here:
`FAILED tests/components/climate/test_device_trigger.py::test_get_triggers - AttributeError: module 'homeassistant.components.climate.const' has no attribute 'HVAC_MODE_COOL'`

--- a/tests/components/climate/test_device_trigger.py
+++ b/tests/components/climate/test_device_trigger.py
@@ -48,7 +48,7 @@ async def test_get_triggers(
     )
     hass.states.async_set(
         entity_entry.entity_id,
-        const.HVAC_MODE_COOL,
+        HVACMode.COOL,
         {
             const.ATTR_HVAC_ACTION: HVACAction.IDLE,
             const.ATTR_CURRENT_HUMIDITY: 23,

